vendor/github.com/OpenBazaar/multiwallet/service/wallet_service.go

```diff
@@ -124,9 +124,12 @@ func (ws *WalletService) listen() {
 		txChan    = ws.client.TransactionNotify()
 		blockChan = ws.client.BlockNotify()
 	)
+
+	var listenAddrs []btcutil.Address
 	for _, sa := range addrs {
-		ws.client.ListenAddress(sa.Addr)
+		listenAddrs = append(listenAddrs, sa.Addr)
 	}
+	ws.client.ListenAddresses(listenAddrs...)
```

Comment:
What changed that we have to unpack the `addrs` from `ws.getStoredAddresses()` now but we didn't before?

Reply:
I don't believe that's true. It unpacked each one as it iterated through the addrs slice and sent it to ListenAddress() individually.

Reply:
Does this satisfy your question @placer14 ?

Reply:
Yes, thanks.